### ISSUE TITLE
Compatibility problem with webtrees 2.1.8

### ISSUE THREAD

**Issue by hartenthaler** [jon48/webtrees-geneajaubart#42]
```
Fatal error in module: myartjaub_admintasks
ReflectionException: Class "MyArtJaub\Webtrees\Module\AdminTasks\AdminTasksModule" does not exist in /mnt/web109/a3/99/51547399/htdocs/hermann/hartenthaler/php/gj/vendor/illuminate/container/Container.php:877 Stack trace: #0
```

It is not important to me, I can wait until you have adopted your version to the new webtrees release.

**Comment by jon48** [comment]
Hello Hermann,

As mentioned in the [upgrade section](https://github.com/jon48/webtrees-geneajaubart#upgrading), you cannot use the automatic upgrade with webtrees-geneajaubart, as that would restore the standard webtrees, and indeed break the tight dependency that we discussed in #8 between most of the MyArtJaub modules and webtrees-geneajaubart.

In effect, webtrees-geneajaubart is like an independent fork of webtrees, with its own release cycle. I have toyed with the idea of disabling or superseding the `UpgradeService` component, but ideally, I would need to request a few tweaks to Greg to make it clean.

I have not had a look yet at the effort to upgrade, but hopefully that is fairly straightforward and can provide something this week.

**Comment by hartenthaler** [comment]
Yes! I know! But last night it was too late and I ignored my hidden knowledge when updating several webtrees installations. Thank you for your kind comment.

**Comment by hartenthaler** [comment]
Just installed and it is working again. Thank you. There are about 50 custom modules available for webtrees (see https://wiki.genealogy.net/Webtrees_Handbuch/Anleitung_für_Webmaster/Erweiterungsmodule#Übersicht_über_bekannte_Erweiterungsmodule). Do you use some of them together with webtrees-geneajaubart?

**Comment by jon48** [comment]
I have tested some of them over the years, and keeping an eye on some specific ones, but I am not using any of them permanently on my own website.

I therefore cannot confirm which ones work nicely with webtrees-geneajaubart. There should not be any major incompatibilities, but if some modules overwrite any views that I am modifying, then some features may be lost in my modules.

**Comment by hartenthaler** [comment]
I just activated the Vesta modules. They are modifying the webtrees core intensely. But as far as I can see, they are working with your modification - besides some colours in the tab "Facts and events". And I have activated the JustLight theme, and again it looks nice.
